api/config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
It's great to have a config file, just be aware that this file could potentially have sensitive info, like DB connection details. Usually you would like to submit to the repo an empty version of this file, and just replace the values when you want to run it on different envs